test/change.test.js

```diff
@@ -610,71 +610,66 @@ describe('Change', function() {
 });
 
 describe('Change with with custom properties', function() {
-  var ChangeWithCustomProperty, TestModelWithModifiedChangeModel;
-  var customProperty = 'tenantId';
+  let Change, TestModel;
 
   beforeEach(function() {
-    var memory = loopback.createDataSource({
+    let memory = loopback.createDataSource({
       connector: loopback.Memory,
     });
 
-    var additionalChangeModelProperties = {};
-    additionalChangeModelProperties[customProperty] = {type: 'string'};
-
-    TestModelWithModifiedChangeModel = loopback.PersistedModel
-    .extend('ChangeTestModel',
+    TestModel = loopback.PersistedModel.extend('ChangeTestModelWithTenant',
```

Comment:
Since we are already within "Change with with custom properties" block, I feel there is no need to repeat that information in model property names.

It is important to use a unique model name in `extend()` arguments though.